code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -1066,3 +1063,63 @@
 		M.u_equip(embedded_id, src, FALSE, TRUE)
 	else
 		embedded_id.forceMove(src)
+
+/// Handles all the locking and unlocking of bracers.
+/obj/item/clothing/gloves/yautja/hunter/verb/toggle_lock()
+	set name = "Toggle Bracer Lock"
+	set desc = "Toggle the lock on your bracers, allowing them to be removed."
+	set category = "Yautja.Misc"
+	set src in usr
+
+	if(usr.stat)
+		to_chat(usr, SPAN_WARNING("You can't do that right now..."))
+		return
+	if(!HAS_TRAIT(usr, TRAIT_YAUTJA_TECH))
+		to_chat(usr, SPAN_WARNING("You have no idea how to use this..."))
+		return FALSE
+
+	attempt_toggle_lock(usr, FALSE)
+
+/obj/item/clothing/gloves/yautja/proc/attempt_toggle_lock(mob/user, force_lock)
+	if(!user)
+		return FALSE
+
+	var/obj/item/grab/G = user.get_active_hand()
+	if(istype(G))
+		var/mob/living/carbon/human/victim = G.grabbed_thing
+		var/obj/item/clothing/gloves/yautja/hunter/bracer = victim.gloves
+		if(isspeciesyautja(victim) && !(victim.stat == DEAD))
+			to_chat(user, SPAN_WARNING("You cannot unlock the bracer of a living hunter!"))
+			return FALSE
+
+		if(istype(bracer))
+			if(alert("Are you sure you want to unlock this [victim.species]'s bracer?", "Unlock Bracers", "Yes", "No") == "Yes")
+				if(user.get_active_hand() == G && victim && victim.gloves == bracer)
+					log_interact(user, victim, "[key_name(user)] unlocked the [bracer.name] of [key_name(victim)].")
+					user.visible_message(SPAN_WARNING("[user] presses a few buttons on [victim]'s wrist bracer."),SPAN_DANGER("You unlock the bracer."))
```

Comment:
a do_after would be nice. 
also i strongly recomend either a longer windup or being unable to unlock the bracers from alive nonyautja because i can clearly see stun bolt spam then take away bracers with 0 effort

Reply:
The point of allowing the bracer to be unlocked on humans is to reduce the need to massacre people to recover them, alongside needing to cut off limbs for it. I'm not going to codify things to force/encourage more people to be killed over it.

Reply:
Thats they way its allways been